README.md

```diff
@@ -129,3 +129,28 @@ If you want to use Isaac Gym, use python3.8 (not python3.9)
 - Multi-Teacher Curriculum Design for Sparse Reward Environments [code](https://github.com/kharyal/cogment-verse/)
 
 (please open a pull request to add missing entries)
+
+
+## Docker
+To run cogment verse from docker containers:
+
+1. Build the cogment-verse image. From the project's root, run the command:
+    ```console
+    $ docker build --tag cogment_verse:local .
+    ```
+2. Launch the container services using docker compose using the command:
+    ```console
+    $ docker compose up
+    ```
+    To specify an experiment or change the orchestrator's web endpoint port, set the environment variables `EXP` or `ORCHESTRATOR_WEB_PORT`
+    ```console
+    $ EXP="+experiment=simple_bc/mountain_car" ORCHESTRATOR_WEB_PORT=9000 docker compose up
+    ```
```

Comment:
`ORCHESTRATOR_WEB_PORT` should be defined in the docker compose

Reply:
Ideally it should work in the same way than w/o docker: you launch a deamon for mlflow (and here it can even be better with docker compose using a `up -d`), and then on demand you launch an experiment.

Reply:
Yes you can pass CLI arguments, but they are overriding the existing entrypoint entirely and I need the 
`python -m main run.mlflow_tracking_uri=http://mlflow:3000 services.orchestrator.web_port=${ORCHESTRATOR_WEB_PORT:-9000} services.orchestrator.web_endpoint=http://localhost:${ORCHESTRATOR_WEB_PORT:-9000}`
part to communicate with the mlflow container and with the orchestrator web endpoint.

Reply:
Done